createdisk-library.sh

```diff
@@ -383,3 +383,18 @@ function download_podman() {
       mv podman-remote/windows/podman-${version}/usr/bin/podman.exe  podman-remote/windows
     fi
 }
+
+# As of now sparsify helper is very specific to OCP/OKD kind of bundle where we get the
+# partition for the root label and then mount it with guestfish to cleanup /boot. With
+# microshift vm we are using lvm and guestfish error out during mount
+# mount /dev/sda3 /
+# libguestfs: error: mount: mount exited with status 32: mount: /sysroot: unknown filesystem type 'LVM2_member'
+# There might be other way for guestfish to mount lvm but as of now using a seperate helper is easy.
+function sparsify_lvm() {
```

Comment:
Can't find a comment I thought I had added, feel free to ignore this comment if you have already answered it elsweher :) imo it would make sense to add a `bundle_type` parameter to `sparsify`, and have a `lvm` and a non `lvm` codepath.